src/module.c

```diff
@@ -10385,6 +10386,8 @@ void moduleFireServerEvent(uint64_t eid, int subid, void *data) {
                 moduledata = data;
             } else if (eid == REDISMODULE_EVENT_SWAPDB) {
                 moduledata = data;
+            } else if (eid == REDISMODULE_EVENT_CONFIG_CHANGE) {
```

Comment:
i still think it may be a good idea to name this event just "CONFIG", and have the "CHANGE" a sub-event.
so that we can add other config related sub-events later.
not sure what it would be used for, just a feeling that it'll allow some flexibility.
@yossigo WDYT?

Reply:
ok, will make this change too.

Reply:
LGTM.